railties/lib/rails/generators/actions.rb

```diff
@@ -220,10 +220,10 @@ def initializer(filename, data = nil)
       #
       #   generate(:authenticated, "user session")
       def generate(what, *args)
-        log :generate, what
```

Comment:
This changes the log format a bit. 
(This will change to `rails` what was indicated as` generate`)  

Is this intentional?

Reply:
@y-yagi I restored the previous logging :+1: